actors/market/src/lib.rs

```diff
@@ -1550,8 +1550,19 @@ fn balance_of(rt: &impl Runtime, owner: &Address) -> Result<TokenAmount, ActorEr
 // Calculates the first update epoch for a deal ID that is no sooner than `earliest`.
 // An ID is processed as a fixed offset within each `interval` of epochs.
 pub fn next_update_epoch(id: DealID, interval: i64, earliest: ChainEpoch) -> ChainEpoch {
-    let q = QuantSpec { unit: interval, offset: id as i64 % interval };
-    q.quantize_up(earliest)
+    // Same logic as QuantSpec from the miner actor, but duplicated here to avoid unnecessary
+    // dependencies.
```

Comment:
we don't have a logical shared place over here to put this so we can dedupe? fil_actors_runtime?

Reply:
because actors/miner/src/quantize.rs isn't a whole lot more than this logic

Reply:
IMO, it's not worth it. Honestly, I'd like to trim down the runtime as well if possible (kind of a grab-bag at this point).